swagger.yaml

```diff
@@ -112,7 +233,7 @@ paths:
         200:
           description: "Categories for area type returned"
           schema:
-            $ref: '#/definitions/GetAreaTypeResponse'
+            $ref: '#/definitions/GetAreaResponse'
```

Comment:
This probably should return a `GetAreasResponse`

Reply:
This should now be updated